### PR TITLE
Add new data source gitlab_group_membership

### PR DESCRIPTION
**Pull request by ilpianista**
As per subject.

### CODE REVIEW

**Comment by vymarkov** [comment]
@ilpianista hi, thanks for awesome work ! 

Are you going to finish this PR in near time?

**Comment by ilpianista** [comment]
> @ilpianista hi, thanks for awesome work !
> 
> Are you going to finish this PR in near time?

Hi, what's missing from your point of view?

**Comment by vymarkov** [comment]
> > @ilpianista hi, thanks for awesome work !
> > Are you going to finish this PR in near time?
> 
> Hi, what's missing from your point of view?

Hi, looks like there were issues with tests, now everything is ok, I hope it will be merged very soon :)

**Comment by TheMacStack** [comment]
This would likely solve a painful issue we are facing with trying to automate changing of users sync'd to a gitlab.com group via SCIM from guest to dev or similar on child projects, is there any knowing when it might be merged?

**Comment by ilpianista** [comment]
Let's try by pinging @ringods.

**Review by armsnyder — CHANGES_REQUESTED** [review]
Hey @ilpianista it looks like your new doc page is not added to the index file (website/gitlab.erb). Can you add it?

**Comment by ilpianista** [comment]
@armsnyder done, thanks!

**Review by armsnyder — APPROVED** [review]
Thanks!

File: website/gitlab.erb [hunk 1]
```diff
@@ -16,6 +16,9 @@
                 <li<%= sidebar_current("docs-gitlab-data-source-group") %>>
                     <a href="/docs/providers/gitlab/d/group.html">gitlab_group</a>
                 </li>
+                <li<%= sidebar_current("docs-gitlab-data-source-group-membership") %>>
+                    <a href="/docs/providers/gitlab/d/group_membership.html">gitlab_group membership</a>
```
Comment (armsnyder) [inline website/gitlab.erb:20]: Looks like a typo here 
```suggestion
                    <a href="/docs/providers/gitlab/d/group_membership.html">gitlab_group_membership</a>
```

**Review by armsnyder — CHANGES_REQUESTED** [review]

**Review by armsnyder — APPROVED** [review]

**Pull request merged**